### DISCUSSION TITLE
Reopened issue remains closed

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#9421] [Mobile] [answered]
I reopened <https://github.com/dotnet/dotnet-api-docs/issues/7080> using GitHub 1.44.0 on iOS 15.2, but when I look at the issue using Safari, it is still closed and there is no trace of me having ever reopened it.

![Popup menu in GitHub app with Reopen item available](https://user-images.githubusercontent.com/46201428/147736792-94eb6c29-f0ff-4b4e-9510-22c20749bbae.png)

![GitHub app shows that I reopened the issue](https://user-images.githubusercontent.com/46201428/147736816-d28689cd-97bf-4fe1-886d-459a33ef84bd.png)

![Safari (not logged in) shows that the issue is still closed and has not been reopened](https://user-images.githubusercontent.com/46201428/147736909-87918959-bc7e-4dbd-93f7-8da205a27e63.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the heads up on this @KalleOlaviNiemitalo — are you able to consistently reproduce this behavior? Or was it a one-time bug? If you are unable to re-open an issue consistently I'll make sure this gets flagged. Right now I'm not able to reproduce this on my end.

**Reply by KalleOlaviNiemitalo** [reply]
I tried reopening it just now, and it is still closed.

**Reply by brianlovin** [reply]
Okay, our team is investigating what might be going on here, thanks!

**Reply by KalleOlaviNiemitalo** [reply]
I wonder if the firewall here is involved — does the app show the "reopened" message even if it cannot reach the server?

**Reply by KalleOlaviNiemitalo** [reply]
It looks like there are some denied attempts to contact firebaselogging-pa.googleapis.com

**Reply by brianlovin** [reply]
@KalleOlaviNiemitalo I think we found a bug where people who don't have permission to reopen an issue are seeing the button in the app to perform the action. We optimistically show the reopened status in the view, even though it did not persist. I'll flag this for our team and investigate.

**Reply by KalleOlaviNiemitalo** [reply]
Seems to be fixed now with GitHub 1.49.1; the app no longer lets me reopen the issue.
